duckduckgo/macos-browser
# Autoconsent refactoring

Task/Issue URL: https://app.asana.com/0/72649045549333/1202153914495789/f
Tech Design URL: https://app.asana.com/0/481882893211075/1202168807850889/f
CC:

**Description**:

Autoconsent library has been rewritten, so this is a new integration in the macos browser
- This is my first time working with Swift, so feel free to be extra picky!
- I will need to bump the autoconsent version after https://github.com/duckduckgo/autoconsent/pull/17, so please don't merge this just yet



**Steps to test this PR**:
1. Build
2. Navigate to any site in the attached list: [testsites.txt](https://github.com/more-duckduckgo-org/macos-browser/files/9121819/testsites.txt)
3. Observe the autoconsent magic (or not)

**Testing checklist**:

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
